workloads/templates/workload-fio-script-cm.yml.j2

```diff
@@ -0,0 +1,244 @@
+        echo "infra" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+      worker_nodes=`oc get nodes -l pbench_agent=true,node-role.kubernetes.io/worker= --no-headers | awk '{print $1}'`
+      for node in $worker_nodes; do
+        echo "worker" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+    fi
+    source /opt/pbench-agent/profile
+    echo "$(date -u) Done configuring pbench for FIO I/O scale test"
+    # End pbench Configuration
+    #
+    # Test Configuration
+    echo "$(date -u) Creating pods for FIO I/O scale test - pods will be in namespace:" {{ fiotest_basename }}0
+    pbench-user-benchmark --config="{{ fiotest_prefix }}-pods-{{ fiotest_maxpods }}-sc-{{ fiotest_storageclass }}-create_pods" -- 'VIPERCONFIG=/root/workload/fiotest.yml openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster should load the cluster [Suite:openshift]"'
+    echo "$(date -u) Pods for FIO I/O test created."
+
+    # wait until all pods are started and then collect data
+    while [[ $(oc get pods -n {{ fiotest_basename }}0 | grep fiotest-pod | grep Run |wc -l) -lt {{ fiotest_maxpods }}  ]] ; do
+      sleep 30
+      echo "Waiting on pods to start..."
+    done
+    ## necessary for user diffrent than root
+    clients=`oc get pods --output=json -n {{ fiotest_basename }}0 | jq -r '[.items[].status.podIP] | join(",")'`
+    mv ~/.ssh/config ~/.ssh/config.bak || true
+    for host in $(echo "${clients}" | sed "s/,/ /g");
+    do
+      HOST=${host} envsubst < /root/workload/ssh_config.template >> ~/.ssh/config
+    done
+    cat ~/.ssh/config.bak >> ~/.ssh/config || true
+    chmod 0600 ~/.ssh/config
+    cp /root/workload/fiotest.sh /tmp/
+
+    # Execute fio test
+    /tmp/fiotest.sh
+    pbench-copy-results --prefix "{{ fiotest_prefix }}"-"{{ fiotest_maxpods }}"-"{{ fiotest_storageclass }}"
+    if [[ {{ fiotest_cleanup }} == "true" ]]; then
+       oc delete project {{ fiotest_basename }}0
+       while [ "$(oc get project |grep {{ fiotest_basename }}0 | awk '{print $1}')" == {{ fiotest_basename }}0 ]; do
+         echo "waiting on project {{ fiotest_basename }}0 to disappear ..."
+         sleep 10
+       done
+       echo "Project {{ fiotest_basename }}0 is deleted ... test finished"
+    elif [[ {{ fiotest_cleanup }} == "false" ]]; then
+       echo "Test is done, but project {{ fiotest_basename }}0 is not be deleted due to FIOTEST_CLAENUP=false"
+    fi
+  fiotest.sh: |
+    #!/bin/bash
+    cp /root/.kube/config /tmp/config
+    export KUBECONFIG=/tmp/config
+    # Not sure is this necessary - re-test
+    sed -i 's/\-o StrictHostKeyChecking=no/\-o StrictHostKeyChecking=no \-l default \-p 2022/g' /opt/pbench-agent/base
+    # to do - fix pbench-fio to support custom options
+    # open issue why getconf.py returns empty string in container
+    curl -o /opt/pbench-agent/bench-scripts/pbench-fio https://raw.githubusercontent.com/ekuric/openshift/master/iotest/pbench-fio
```

Comment:
The container has `pbench-fio` installed, but over-writing the `pbench-fio` script seems a bit over kill just to get the `pbench-fio` version string.  That value is supposed to come from the `pbench-agent.cfg` file.

Every pbench-agent install has an `/opt/pbench-agent/config/pbench-agent.cfg` file.  So if this container image does not have that configuration file installed with a valid `pbench-fio` version string, then something is wrong with how the containers are built.

If it *does* have a `pbench-agent.cfg` file, then there is something else wrong with the installation of pbench-agent in the container.  Where is that install performed?  I assume when the container is built ...

Reply:
Seems like you could add a test at line 34 about to ensure the existence of the `pbench-agent.cfg` file.

Reply:
pbench-fio was installed inside container with `yum pbench-fio -y ` during image build , so I expect it set all fine as for any case where pbench-fio is installed , will check for `/opt/pbench-agent/config/pbench-agent.cfg`

Reply:
This is fixed if `/workloads/templates/pbench-cm.yml.j2` is updated and added 
```
 [pbench-fio]
 version = 3.12
 histogram_interval_msec = 10000
```
and later in this script pointed to this new file `export CONFIG=/opt/pbench-agent/config/pbench-agent.cfg`